src/components/JourneyDisplay.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { GiSnail } from 'react-icons/gi';
+import { HiArrowCircleRight, HiClock } from 'react-icons/hi';
+import { SiPuma } from 'react-icons/si';
+import StationDisplay from './JourneyDisplayComponents/StationDisplay';
+import StatusDisplay from './JourneyDisplayComponents/StatusDisplay';
+import TimeDisplay from './JourneyDisplayComponents/TimeDisplay';
+import { JourneyInfo } from './MainPage';
+
+interface JourneyDisplayArgs {
+    journeyInfo: JourneyInfo;
+}
+
+const JourneyDisplay: React.FC<JourneyDisplayArgs> = ({ journeyInfo }) => {
+
+    const minutesToHoursAndMinutes = (minutes : number): string => {
+        if (minutes < 60) {
+            return minutes + ' min';
+        } else {
+            const hours  = Math.floor(minutes / 60);
+            const remainingMinutes = minutes % 60;
+            return hours + 'h ' + remainingMinutes + 'min';
+        }
+    };
+
+    return (
+        <div className = "journey-display" 
+            style = { { backgroundColor: journeyInfo.isFastestJourney ? '#bde8ae' : journeyInfo.isOvertaken ? '#e2db6f' : 'white' } }>
+
+            {journeyInfo.isFastestJourney && <div className = 'fastest-train-tag'><SiPuma/>Fastest Train</div>}
+            {journeyInfo.isOvertaken && <div className = 'slowest-train-tag'><GiSnail/> Slow Train</div>}
+
+            <div className = 'first-row'>
```

Comment:
What does first-row mean? Do you think there might be a clearer name which matches the "duration-display" naming convension

Reply:
renamed to "status-and-route"